services/wallet/saved_addresses_test.go

```diff
@@ -14,49 +13,50 @@ import (
 	"github.com/status-im/status-go/sqlite"
 )
 
+const (
+	ensMember int = iota
+	isTestMember
+	addressMember
+)
+
 func setupTestSavedAddressesDB(t *testing.T) (*SavedAddressesManager, func()) {
-	tmpfile, err := ioutil.TempFile("", "wallet-saved_addresses-tests-")
-	require.NoError(t, err)
-	db, err := appdatabase.InitializeDB(tmpfile.Name(), "wallet-saved_addresses-tests", sqlite.ReducedKDFIterationsNumber)
+	db, err := appdatabase.InitializeDB(sqlite.InMemoryPath, "wallet-saved_addresses-tests", sqlite.ReducedKDFIterationsNumber)
```

Comment:
@IvanBelyakoff why did you change this?

Not sure, but maybe this could work as well:
```
func setupTestSavedAddressesDB(t *testing.T) (* SavedAddressesManager, func()) {
	db, stop, err := appdatabase.SetupTestSQLDB("settings-tests-")
	if err != nil {
		require.NoError(t, stop())
	}
	require.NoError(t, err)

        return &SavedAddressesManager{db}, func() {
		require.NoError(t, stop())
	}
}
```

Reply:
Great thinking @IvanBelyakoff. Usually, the I/O is much slower than the memory, which should provide considerable gains in the development environment and a good speed up in the CI. Most of the tests, should not be required (for sure the ones that don't deal with user folder management).

Let's bring this matter to the `status-go` channel and find out if anybody notices any issue with bringing this improvement to all tests (`SetupTestSQLDB ` and general setup entries)